README.md

```diff
@@ -6,20 +6,20 @@
 
 The service-catalog project is in incubation to bring integration with service
 brokers to the Kubernetes ecosystem via the [Open Service Broker
-API](https://github.com/openservicebrokerapi/servicebroker). A service broker
-is an endpoint that manages a set of services.  The end-goal of the service-
-catalog project is to provide a way for Kubernetes users to consume services
-from brokers and easily configure their applications to use those services,
-without needing detailed knowledge about how those services are created /
-managed.
+API](https://github.com/openservicebrokerapi/servicebroker). A service
+**broker** is an endpoint that manages a set of **service classes**.  To use a
+service class, a user *provisions* an **instance** of that service.  Some
+services have the ability to inform users of how they should be consumed in
+applications.  Users create **bindings** to be injected with information about
+how a service instance should be consumed.
```

Comment:
perhaps instead of "should be consumed" we should talk about how the binding info is used to convey the coordinated and connection info about it. The word "consumed" could be misleading, I think.